### PR TITLE
add dmsassets folder to file system sync exclusion, fixes #190

### LINKED ISSUE #190: dms documents get synced into assets table
the DMS documents show up in the assets table after a file system sync. even though the general underscore folder doesn't show in the CMS, the files still get synced into the table because the sync exclusion on https://github.com/silverstripe/silverstripe-framework/blob/3.6/filesystem/Filesystem.php#L35-L41 doesn't exclude all underscore folders.

### PR DESCRIPTION
**Pull request by xini**

### CODE REVIEW

File: _config.php [hunk 1]
```diff
@@ -19,3 +19,9 @@
     //using the same db relations for the versioned documents, as for the actual documents
     $config->update('DMSDocument_versions', 'db', $config->get('DMSDocument', 'db'));
 }
+
+// add dmsassets folder to file system sync exclusion
+if (strpos($config->get('DMS', 'folder_name'), 'assets/') === 0) {
+    $folderName = str_replace('assets/', '', $config->get('DMS', 'folder_name'));
```
Comment (robbieaverill) [inline _config.php]: Maybe we could do `substr($config->get('DMS', 'folder_name'), 0, 7)` instead? I have a feeling this wouldn't work if the folder name was a subfolder like "assets/my-awesome-assets/some-other-folder" for example

**Comment by xini** [comment]
@robbieaverill you're right of course. I have made the change. thanks!

**Comment by robbieaverill** [comment]
Thanks @xini!

**Comment by xini** [comment]
thank you!

**Pull request merged**